solutions/izho-17-LongestBeautifulSequence.mdx

```diff
@@ -0,0 +1,195 @@
+---
+id: izho-17-LongestBeautifulSequence
+source: IZhO 2017
+title: 2017 - Longest beautiful sequence
+author: Shijie Ren
+---
+
+<Spoiler title="Hint 1">
+
+Both $\mathcal{O}(N^2)$ and $\mathcal{O}(N\max{a[i]})$ algorithms are not hard
+to come up with, but the latter one is more enlightening.

```

Comment:
please elaborate
not everyone is as smart as you

Reply:
also, what does "more enlightening" mean?

Reply:
ok, but they're repeated in hint2, and their equations are given in the explanation

Reply:
> also, what does "more enlightening" mean?

I want to express n2 solution can't be optimized further, maybe "more enlightening" is meaningless

Reply:
; )

Reply:
> ok, but they're repeated in hint2, and their equations are given in the explanation

then move them here, just saying that they exist does basically nothing